docs/apache-airflow-providers-http/operators.rst

```diff
@@ -37,6 +37,13 @@ Here we are poking until httpbin gives us a response text containing ``httpbin``
     :start-after: [START howto_operator_http_http_sensor_check]
     :end-before: [END howto_operator_http_http_sensor_check]
 
+This sensor can also be used in deferrable mode
+
+.. exampleinclude:: /../../tests/system/providers/http/example_http.py
+    :language: python
+    :start-after: [START howto_operator_http_http_sensor_check_async]
```

Comment:
```suggestion
    :start-after: [START howto_operator_http_http_sensor_check_deferrable]
```

I might suggest changing it to `deferrable` to make the wording consistent

Reply:
updated